pkg/model/components/apiserver.go

```diff
@@ -155,8 +155,10 @@ func (b *KubeAPIServerOptionsBuilder) BuildOptions(o interface{}) error {
 		c.EnableAdmissionPlugins = append(c.EnableAdmissionPlugins, c.AppendAdmissionPlugins...)
 	}
 
-	// We make sure to disable AnonymousAuth
-	c.AnonymousAuth = fi.Bool(false)
+	// We default AnonymousAuth to off
```

Comment:
Perhaps have ability to enable `AnonymousAuth` under the `PublicJWKS` feature flag?